help/md/fnJC.md

```diff
@@ -1,9 +1,23 @@
 ## name
 fnJC
 ## title
+fnJC
 ## description
+Jukes cantor transition matrix
 ## details
+Transition matrix with n amount of states in which each state has an equal
+probability of to change to any other state. The only parameter in this model
+is mu with the rate of change from any given state being mu / number of states.
```

Comment:
Saying that mu is a parameter could be a bit confusing because the user doesn't need to pass mu as a function argument.